packages/components/src/components/listitem/listitem.styles.ts

```diff
@@ -96,12 +97,24 @@ const styles = css`
   :host::part(trailing-text) {
     text-align: right;
   }
+
+  :host(:dir(ltr))::part(trailing-text-side-header),
+  :host(:dir(ltr))::part(trailing-text-subline) {
+    margin-left: var(--mdc-listitem-column-gap);
+  }
+
+  :host(:dir(rtl))::part(trailing-text-side-header),
+  :host(:dir(rtl))::part(trailing-text-subline) {
+    margin-right: var(--mdc-listitem-column-gap);
+  }
```

Comment:
Please use [logical margin](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline-start) to handle rtl/ltr:


```suggestion
  :host()::part(trailing-text-side-header),
  :host()::part(trailing-text-subline) {
    margin-inline-start: var(--mdc-listitem-column-gap);
  }
```